Cargo.toml

```diff
@@ -35,6 +36,7 @@ sails-idl-parser = { path = "rs/idl-parser" }
 sails-client-gen = { path = "rs/client-gen" }
 sails-rs = { path = "rs" }
 sails-idl-gen = { path = "rs/idl-gen" }
+sails-idl-meta = { path = "rs/idl-meta" }

```

Comment:
Packages here are going in the order of dependencies, i.e.,
this has to be put first as it doesn't depend on any sails packages.
The `sails-idl-gen` has to be put before `sails-rs` as the latter now depends on the former

Reply:
ok, sails-idl-meta going first, before macors